docs/reviewing-release-prs.md

```diff
@@ -0,0 +1,176 @@
+
+You can do this by opening your terminal locally, switching to the release branch (which takes the form `release/X.Y.Z`), and running this command (where `package-name` is the package you're interested in, and `current-version` is the version it has now):
+
+```
+git log --oneline @metamask/<package-name>@<current-version>..HEAD -- packages/<package-name>
+```
+
+This should give you the set of commits that have taken place since the package's latest release which changed the package in any way.
+
+You can also request a diff with each commit:
+
+```
+git log -p @metamask/<package-name>@<current-version>..HEAD -- packages/<package-name>
+```
+
+Or, you can view all of the changes in one:
+
+```
+git diff @metamask/<package-name>@<current-version>..HEAD -- packages/<package-name>
+```
+
+You'll reference these changes in upcoming steps.
+
+## 2. Determine impact of changes
+
+Now that you know what sort of changes are being made, the next step is look through the version bumps in the release PR and verify the impact of those changes so that it can be properly communicated.
+
+First, it's helpful to understand what sort of changes are worth communicating. While a release could certainly ship with a list of every single change made to the code in this repository, this would be needless. The people who will end up using the packages in this monorepo — consumers — only care about that part of code they can actually see and work with. So when it comes to understanding impact, this is what you should pay attention to:
+
+- Changes to interfaces
+- Changes to behavior
+- Changes to runtime dependencies
+- Changes to consumer-facing documentation
+
+Conversely, you generally should not need to worry about:
+
+- Changes to internal tooling
+- Changes to developer-only dependencies
+- Changes to contributor-only documentation
+
+With that in mind, there are four categories of changes that determine the level of impact.
+
+### Breaking changes
+
+If a consumer upgrades a package to a new version without making any modifications to their code, that version is "breaking" if it causes:
+
+- An error at runtime
+- An error at compile time (e.g., a TypeScript type error)
+- An error at install time (e.g., the consumer's package manager reports a Node version incompatibility)
+- A surprising difference in behavior
+
+Given this, there are many ways a change could be regarded as breaking:
+
+- Changing a function or method to throw an error
+- Adding a required argument to a function or method
+- Adding a property to a TypeScript type
+- Narrowing the type of a property in a TypeScript type
+- Narrowing the type of an argument in a function
+- Adding a parameter to a TypeScript type
+- Removing a parameter from a TypeScript type
+- Upgrading a runtime dependency to a version which causes any of the above
+- Removing an export from the package
+- Renaming the package
+- Bumping the Node version
+- Bumping the NPM version
```

Comment:
Hmm, what does this refer to?

Reply:
I believe this came from one of our forked repos where `engines.npm` was already specified in `package.json`, and we bumped it. That said, I've only seen this once and I suspect it will be very rare, so I can remove this.

Reply:
I have removed this item.